src/main/java/com/amazon/opendistroforelasticsearch/knn/plugin/rest/RestKNNStatsHandler.java

```diff
@@ -73,17 +75,35 @@ public String getName() {
         return NAME;
     }
 
+
+    private List<String> getStatsPath() {
+        List<String> statsPath = new ArrayList<>();
+        statsPath.add("/{nodeId}/stats/");
+        statsPath.add("/{nodeId}/stats/{stat}");
+        statsPath.add("/stats/");
+        statsPath.add("/stats/{stat}");
+        return statsPath;
+    }
+
+    private Map<String, String> getUrlPathByLegacyUrlPathMap() {
```

Comment:
I was spitballing the name, but since we're here, getUrlPath**To**LegacyUrlPathMap?

Reply:
@dblock Here, map's key is Legacy and value is Latest. Hence, i added used by to represent getValueByKey format. What do you think?

Reply:
A-OK